react-es6-popup/webpack.config.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: {
+    // Each entry in here would declare a file that needs to be transpiled
+    // and included in the extension source.
+    // For example, you could add a background script like:
+    // background: './src/background',
+    popup: './src/popup',
+  },
+  output: {
+    // This copies each source entry into the extension dist folder named
+    // after its entry config key.
+    path: 'extension/dist',
+    filename: '[name].js',
+  },
+  module: {
+    // This transpiles all code (except for third party modules) using Babel.
+    loaders: [{
+      exclude: /node_modules/,
+      test: /\.js$/,
+      // Babel options are in .babelrc
+      loaders: ['babel'],
+    }],
+  },
+  resolve: {
+    // This allows you to import modules just like you would in a NodeJS app.
+    extensions: ['', '.js', '.jsx'],
```

Comment:
I'm not totally sure why but if I remove it, webpack gives me this error:

ERROR in ./~/react-dom/lib/flattenChildren.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../process/browser.js in /Users/kumar/dev/webextensions-examples/react-es6-popup/node_modules/react-dom/lib